src/test/java/it/clients/symphony/api/StreamsClientTest.java

```diff
@@ -485,4 +487,14 @@ public void getUserStreamsSuccess() {
     assertNotNull(streamsList);
     assertEquals(1, streamsList.size());
   }
+
+  @Test(expected = IllegalArgumentException.class)
+  public void failToGetUserStreamsWithIllegalSkipValue() {
+    this.streamsClient.getUserStreams(Collections.emptyList(), true, -1, 0);
+  }
+
+  @Test(expected = IllegalArgumentException.class)
+  public void failToGetUserStreamsWithIllegalLimitValue() {
+    this.streamsClient.getUserStreams(Collections.emptyList(), true, 0, -1);
+  }
```

Comment:
Is it possible to add a new test to cover the conventional behavior of the new method?

Reply:
It is actually already covered